contentacms/contenta_jsonapi
# feat: use the Umami content model

Task: #303


Creates the new content model using the Umami profile with one exception: we are keeping the media integration.

### Notes

The migration is currently broken, so DO NOT MERGE.

### Test Instructions

Re-install contenta and check that the content is present using the new content model.
